v0/destinations/posthog/data/PHGroupConfig.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+      "destKey": "timestamp",
+      "sourceKeys": ["originalTimestamp", "timestamp"],
+      "required": false
+    },
+  { "destKey": "groupId", "sourceKeys": "groupId", "required": false },
```

Comment:
the group call here is inking group to a user.
Is it also possible to create/update the group itself in Posthog?

Reply:
Posthog provide a feature called Cohort where we can create a group in Posthog and link a particular user to that group.

Reply:
as discussed